templates/database.yml.epp

```diff
@@ -30,6 +30,6 @@
   username: <%= $username %>
 <% } -%>
 <% if $password { -%>
-  password: "<%= $password %>"
+  password: <%= stdlib::to_ruby($password) %>
```

Comment:
If I drop this, I get:
```
expected that the catalogue would contain File[/etc/foreman/database.yml] with content set to /password: "secret"/ but it is set to #<Sensitive [value redacted]>
```

so maybe `with_content` is a bad idea to begin with?

Reply:
`with_content(sensitive(…))` works for the plain case, but not once to_ruby is involved. yay